### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by rhcarvalho**
As per email conversation, tentative improvements:

1. Use `openshift/` dir for OS-related files, in particular `openshift/templates/` for templates.
2. Name the template files in a concise and descriptive way.
3. Document fact that PostgreSQL db in the template has ephemeral storage
4. Document how to read logs

There's more to be done, just want to keep pushing early and often :trophy:

### CODE REVIEW

**Comment by rhcarvalho** [comment]
@bparees PTAL

**Comment by bparees** [comment]
can you add some doc around the run-in-container.sh script to the README?  moving what you have in the script header might be sufficient, i just think it would be more useful in the readme.

otherwise looks good.

**Comment by rhcarvalho** [comment]
@bparees, done. I've moved the section about repo structure up, and documented the usage of the script in the README.

**Comment by bparees** [comment]
awesome, thanks...squash and i'll merge it.

**Comment by rhcarvalho** [comment]
@bparees I've squashed the improvements.

Included new assemble and run scripts for early appreciation before placing them with tests in sti-python. Yes, they will be removed from this repo soon.

The new scripts fix #5.

File: openshift/templates/django-postgresql.json [hunk 1]
```diff
@@ -328,18 +324,12 @@
...
     {
-      "name": "APP_MODULE",
-      "description": "Python dotted path to your Django WSGI application",
-      "value": "project.wsgi"
-    },
-    {
       "name": "APP_CONFIG",
-      "description": "Relative path to Gunicorn configuration file (optional)",
-      "value": "gunicorn_conf.py"
+      "description": "Relative path to Gunicorn configuration file (optional)"
```
Comment (bparees) [inline openshift/templates/django-postgresql.json]: @jcantrill fyi here's a parameter definition with no value and no generator.  so i think we made the right call in treating it as acceptable.

File: .sti/bin/assemble [hunk 2]
```diff
@@ -1,52 +1,63 @@
...
 
-if [ -f setup.py ]; then
-    echo "---> Installing application ..."
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    echo "---> Installing dependencies ..."
-    pip install --user -r requirements.txt
-fi
+  echo "---> Copying application source ..."
+  cp -Rf /tmp/src/* ./
```
Comment (bparees) [inline .sti/bin/assemble]: i know you didn't add this and we do it in all our scripts, but as a rule copying to WORKDIR really bothers me.  Seems like we ought to be explicit about where we're putting the app source.  @mfojtik @soltysh was there a reason we used this pattern that i'm forgetting?

**Comment by bparees** [comment]
lgtm.

File: .sti/bin/assemble (hunk 2 above)
Comment (soltysh) [inline .sti/bin/assemble]: I remember complaining about not being explicit, like it used to be when
that was introduced, sometime when we created the base image. I'm +1000 to
make it explicit back. @mfojtik gave me some reasoning behind that change
but I don't remember what was that.

**Pull request merged**